### PR TITLE
add slash storage clear and account balance update

### PR DESCRIPTION
**Pull request by vedhavyas**
This PR adds a migration:
- Remove slash data
- Add 63 rad balance to the CNFProxy account

### CODE REVIEW

File: runtime/src/lib.rs [hunk 1]
```diff
@@ -891,8 +891,41 @@ pub type SignedPayload = generic::SignedPayload<Call, SignedExtra>;
...
+            // This will remove all the slash related data on staking pallet
+            clear_slash_data::<Runtime>();
+            // This is the total balance we lost
+            let balance = 63 * RAD;
+            // this is the centrifuge CNF proxy account: 4fsNBXAXoUKxSeu9qKCD9s285dzjS7Nmh1u76s4AsidrdmdL
+            let acc :AccountId = hex!("c84b10ac8baea482e93d961a52502c9ee8227a7deaffd746657c5a8655cc70ab").into();
+            let res :DispatchResult = Balances::add_balance(acc, balance);
+            match res{
+                Ok(()) => {
+                    sp_runtime::print("Account balance updated");
```
Comment (mikiquantum) [inline runtime/src/lib.rs:915]: These logs are helpful 👍

File: runtime/src/lib.rs [hunk 2]
```diff
@@ -891,8 +891,41 @@ pub type SignedPayload = generic::SignedPayload<Call, SignedExtra>;
...
+    use frame_support::{traits::OnRuntimeUpgrade, weights::Weight};
+    use pallet_staking::migrations::clear_slash_data;
+    use hex_literal::hex;
+    use sp_runtime::{DispatchResult};
+
+    pub struct Upgrade;
+    impl OnRuntimeUpgrade for Upgrade {
+        fn on_runtime_upgrade() -> Weight {
+            // This will remove all the slash related data on staking pallet
+            clear_slash_data::<Runtime>();
```
Comment (mikiquantum) [inline runtime/src/lib.rs:907]: If at the time of the upgrade, there are some unapplied slashes they will be deleted as well, right? No objection, just informational.

**Review by mikiquantum — APPROVED** [review]
Looks good, thanks!

File: runtime/src/lib.rs (hunk 2 above)
Comment (vedhavyas) [reply runtime/src/lib.rs:907]: yeap this will remove all the data related to slash

**Pull request merged**